src/reset.scss

```diff
@@ -0,0 +1,23 @@
+@import './variables.scss';
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+}
+
+button {
+  border: none;
+  background: none;
+  padding: 0;
+  cursor: pointer;
```

Comment:
The button does not have the right font anymore, because `button` elements do not inherit the font.

Reply:
oh this comment was supposed to go to the AmbassadorButton component
https://github.com/alveusgg/extension/blob/8f007cbc9a7ce9c8f0bd539cab35c789508a452e/src/utils/global/ambassadorButton/AmbassadorButton.tsx#L19

Reply:
Updated the reset to include an inherit for font-family